client/README.md

```diff
@@ -94,18 +86,23 @@ commands. This command will run galaxy without building the client:
 
     make skip-client
 
+Or by setting the following environment variable and running Galaxy however you
+prefer:
+
+    GALAXY_SKIP_CLIENT_BUILD=1 ./run.sh
+
 ## Changing Styles/CSS
 
-Galaxy uses Sass for its styling, which is a superset of CSS that compiles down
-to regular CSS. Most Galaxy styling source (.scss) files are kept in
-`client/src/style/scss`. There are additionally style blocks alongside some Vue
-components -- styles that are particular to that individual component and do
-not apply site-wide.
+Galaxy uses Sass for globally applied styling, which is a superset of CSS that
+compiles down to regular CSS. Most Galaxy styling source (.scss) files are kept
+in `client/src/style/scss`. Many components will also have local style blocks
+containing styles that are particular to that individual component and do not
+apply site-wide.
 
 On build, the compiled css bundle is served at `/static/style/base.css`.
 
-As mentioned above, `make client` will rebuild styles, as a part of the webpack
-build. For iterative development, "Watch Mode" rebuilds as described above do
+As mentioned above, `make client` will rebuild styles as a part of the webpack
+build. For iterative development "Watch Mode" rebuilds as described above do
```

Comment:
I think we should keep the comma after `For iterative development`, since it's like saying "When doing iterative development, ..."